packages/core/lib/cliHelp.js

```diff
@@ -0,0 +1,33 @@
+const handleHelpInput = function ({ inputStrings }) {
+  let displayHelp = false;
+  //User only enter truffle with no commands, let's show them what's available.
+  if (inputStrings.length === 0) {
+    displayHelp = true;
```

Comment:
The code can return `true` here.

Reply:
No it can't, that does not conform to the method signature as I wrote it :)

Reply:
oh do you mean to return `{ displayHelp: true }` in both of these cases?

Reply:
Yyes, `{ displayHelp: true } ` in both of those cases.

Reply:
Sure but that is more a stylistic thing than anything

Reply:
I consider it to be a readability refactor. The current iteration makes it easier to understand the success path of the function. It reduces the need to track `displayHelp`'s state throughout the function, as well as clearly outlining the exceptional, failure, cases with the early returns.